### PR TITLE
 change news publishing fields to use new Admin Scheduler system 

### PR DESCRIPTION
**Pull request by cebe**
### What are you changing/introducing

Remove timestamp fields and use the admin scheduler instead.

### What is the reason for changing/introducing

Better and more flexible, as discussed in slack today.
This also fixes the issue that timestamp columns where not checked in frontend controller.

### QA

| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | yes
| Breaks BC?    | yes! (need to check if migration can or should be improved)
| Tests pass?   | n/a

### CODE REVIEW

**Comment by cebe** [comment]
this is working as expected now. @nadar ready to merge from my side.

File: src/admin/migrations/m190624_112612_news_article_scheduler.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        $this->execute(<<<SQL
+UPDATE news_article
+SET is_online = (timestamp_display_from < NOW() AND (
+    NOT is_display_limit OR (
+        is_display_limit AND timestamp_display_until > NOW()
+    )
+));
+SQL
+        );
+        // TODO add scheduler entries?
```
Comment (cebe) [inline src/admin/migrations/m190624_112612_news_article_scheduler.php:19]: @nadar you might want to adjust the migration to convert existing timestamps to schedulers....

Comment (nadar) [reply src/admin/migrations/m190624_112612_news_article_scheduler.php:19]: if we release under version 2.0 i don't think it's a problem.

**Review by nadar — CHANGES_REQUESTED** [review]
Looks good to me! Please add a changelog entry under version 2.0 as new migrations are BC breaks. Thank you! I will add an UPGRADE.md text entry.

**Comment by nadar** [comment]
I'll do the changelog entry for you afterwards ;-) thanks

**Pull request merged**